backend/mock/service/githubmock/githubmock.go

```diff
@@ -63,6 +63,10 @@ func (s svc) ListOrganizations(ctx context.Context, user string) ([]*githubv3.Or
 	}, nil
 }
 
+func (s svc) ListPullRequestsWithCommit(ctx context.Context, ref *github.RemoteRef, sha string) ([]*github.PullRequestInfo, error) {
+	panic("implement me")
```

Comment:
can you add a stub for the response? The data from the tests should be ok to use here

Reply:
done